rs/pocket_ic_server/src/pocket_ic.rs

```diff
@@ -1317,22 +1328,72 @@ impl Operation for CallRequest {
                 )
                 .build();
 
-                let svc = CallServiceV2::new_service(ingress_validator);
+                let subnet_clone = subnet.clone();
+
+                // Task that waits for call service to submit the ingress message, and
+                // forward it to the state machine. The task will automatically terminate
+                // once the call service sends a message, or if the call service is dropped.
```

Comment:
```suggestion
                // Task that waits for call service to submit the ingress message, and
                // forwards it to the state machine. The task will automatically terminate
                // once it submits an ingress message received from the call service to the `StateMachine`, or if the call service is dropped (in which case `r.recv().await` fails with an error).
```

Reply:
please double check my suggestion before applying it

Reply:
Changed your suggestion  slightly